src/main/java/de/tum/in/www1/artemis/web/rest/TutorParticipationResource.java

```diff
@@ -0,0 +1,172 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import de.tum.in.www1.artemis.domain.*;
+import de.tum.in.www1.artemis.domain.enumeration.TutorParticipationStatus;
+import de.tum.in.www1.artemis.repository.ExampleSubmissionRepository;
+import de.tum.in.www1.artemis.service.*;
+import de.tum.in.www1.artemis.web.rest.util.HeaderUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static de.tum.in.www1.artemis.web.rest.util.ResponseUtil.forbidden;
+
+/**
+ * REST controller for managing TutorParticipation.
+ */
+@RestController
+@RequestMapping({"/api", "/api_basic"})
+@PreAuthorize("hasRole('ADMIN')")
+public class TutorParticipationResource {
+
+    private final Logger log = LoggerFactory.getLogger(TutorParticipationResource.class);
+    private final TutorParticipationService tutorParticipationService;
+    private final ExerciseService exerciseService;
+    private final CourseService courseService;
+    private final UserService userService;
+    private final ExampleSubmissionRepository exampleSubmissionRepository;
+    private final ExampleSubmissionService exampleSubmissionService;
+
+    public TutorParticipationResource(TutorParticipationService tutorParticipationService,
+                                      CourseService courseService,
+                                      ExerciseService exerciseService,
+                                      UserService userService,
+                                      ExampleSubmissionRepository exampleSubmissionRepository,
+                                      ExampleSubmissionService exampleSubmissionService) {
+        this.tutorParticipationService = tutorParticipationService;
+        this.exerciseService = exerciseService;
+        this.courseService = courseService;
+        this.userService = userService;
+        this.exampleSubmissionRepository = exampleSubmissionRepository;
+        this.exampleSubmissionService = exampleSubmissionService;
+    }
+
+    /**
+     * POST /exercises/:exerciseId/tutorParticipations : start the "id" exercise for the current user.
```

Comment:
This comment is copied and pasted from the other participation call. Please describe a bit more in detail how this is related to tutors only (instead of current user) and what is actually started